.github/workflows/main.yml

```diff
@@ -0,0 +1,313 @@
+          # get current version
+          version=$(pip show qiskit-algorithms | awk -F. '/^Version:/ { print substr($1,10), $2-1 }' OFS=.)
+          # download stable version
+          wget https://codeload.github.com/qiskit-community/qiskit-algorithms/zip/stable/$version -O /tmp/repo.zip
+          unzip /tmp/repo.zip -d /tmp/
+          # copy stable tutorials to main tutorials
+          cp -R /tmp/qiskit-algorithms-stable-$version/docs/tutorials/* docs/tutorials
+          # run tutorials and zip results
+          echo "earliest_version: 0.1.0" >> releasenotes/config.yaml
+          # ignore unreleased/untagged notes
+          tools/ignore_untagged_notes.sh
+          make html
+          cd docs/_build/html
+          mkdir artifacts
+          tar -zcvf artifacts/tutorials.tar.gz --exclude=./artifacts .
+        if: ${{ matrix.python-version == 3.8 && !startsWith(github.ref, 'refs/heads/stable') && !startsWith(github.base_ref, 'stable/') }}
+        shell: bash
+      - name: Run upload stable tutorials
+        uses: actions/upload-artifact@v3
+        with:
+          name: tutorials-stable${{ matrix.python-version }}
+          path: docs/_build/html/artifacts/tutorials.tar.gz
+        if: ${{ matrix.python-version == 3.8 && !startsWith(github.ref, 'refs/heads/stable') && !startsWith(github.base_ref, 'stable/') }}
+  Deprecation_Messages_and_Coverage:
+    needs: [Checks, Algorithms, Tutorials]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - uses: actions/download-artifact@v3
+        with:
+          name: ubuntu-latest-3.8
+          path: /tmp/u38
+      - uses: actions/download-artifact@v3
+        with:
+          name: ubuntu-latest-3.9
+          path: /tmp/u39
+      - uses: actions/download-artifact@v3
+        with:
+          name: ubuntu-latest-3.10
+          path: /tmp/u310
+      - uses: actions/download-artifact@v3
+        with:
+          name: ubuntu-latest-3.11
+          path: /tmp/u311
+      - uses: actions/download-artifact@v3
+        with:
+          name: macos-latest-3.8
+          path: /tmp/m38
+      - uses: actions/download-artifact@v3
+        with:
+          name: macos-latest-3.11
+          path: /tmp/m311
+      - uses: actions/download-artifact@v3
+        with:
+          name: windows-2019-3.8
+          path: /tmp/w38
+      - uses: actions/download-artifact@v3
+        with:
+          name: windows-2019-3.11
+          path: /tmp/w311
+      - name: Install Dependencies
+        run: pip install -U coverage coveralls diff-cover
+        shell: bash
+      - name: Combined Deprecation Messages
+        run: |
+          sort -f -u /tmp/u38/ml.dep /tmp/u39/ml.dep /tmp/u310/ml.dep /tmp/u311/ml.dep /tmp/m38/ml.dep /tmp/m311/ml.dep /tmp/w38/ml.dep /tmp/w311/ml.dep || true
```

Comment:
This is more just a comment. I do not know why the names of these files were not a little more generic. The `ml` aspect is because its machine learning (if you look at the same file in the other app repos the name reflects the app - maybe its in case it was all run in one place the names would not collide). I guess it could be changed to `alg.dep` and `alg.dat` below and elsewhere they are created/referenced, but it will work with `ml` of course, but thats why its ml here